docs/intro_testers.rst

```diff
@@ -31,10 +31,10 @@ Each `job` definition contains a list of `tasks` to execute, along with
 `roles`. `Roles` tell `teuthology` how many nodes to use for each `job` along
 with what functions each node will perform.
 
-To go into more depth regarding suite design, see `ceph-qa-suite`'s `README`.
+To go into more depth regarding suite design, see `ceph/qa`'s `README`.
```

Comment:
The `ceph/qa` is ambiguous, `ceph/qa` is not a repo.

Reply:
Fixed. Better now?